### DISCUSSION TITLE
How can we monitor stack traces in Apache Skywalking?

### DISCUSSION THREAD

**Discussion by SooryaMuthuraj** [apache/skywalking#11729] [General]
How can we monitor stack traces(classes and methods) in Apache Skywalking? Kindly suggest how to perform it.

Regards,
Surya.

**Comment by wu-sheng** [comment]
That is impossible. You can only profile the stack and on demand only.
Tracing stacks always will impact your service performances badly.

**Reply by SooryaMuthuraj** [reply]
Hi,

How can we profile the stack ? please help.

Regards,
Surya

**Reply by wu-sheng** [reply]
Read the docs, or ask AI on the website.
